packages/preview/modern-uit-thesis/0.1.0/modules/abstract.typ

```diff
@@ -0,0 +1,9 @@
+#let abstract_page(body) = {
```

Comment:
The recommend casing convention is kebab-case, so `abstract-page`, though it's not a strict requirement.